src/js/components/initialmessage.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @param {object} data contains the expected page content data
+ * returns a default component to show the provided information.
+ */
+const initialMessageComp = (data) => {
+  const section = document.createElement('section');
+  
+  if (data.backgroundColor) section.classList.add(`text-block-comp__container--${data.backgroundColor}`);
```

Comment:
This component does not have a background color variation. The green box with the white color section is going to be handled in a different ticket.

Reply:
Done!!!